files/teaching-demo-rubric.md

```diff
@@ -36,16 +36,16 @@ Date:
 
 |Aspect|Comments_________________________________________________|
 |------|---------------------|
-|<br>Speed<br>|
-|<br>Voice, teaching style|
-|<br>Verbalize typing|
-|<br>Verbalize thoughts|
-|<br>Explain all that was typed|
-|<br>Easy to follow|
-|<br>font, window size etc|
-|<br>Mirroring Learner’s Environment|
-|<br>Stick to the Material|
-|<br>Embrace Mistakes|
-|<br>Use of Illustrations|
+|<br>Sticks to curriculum<br>|
+|<br>Context|
+|<br>Knowledge|
+|<br>Mirror learner's environment|
+|<br>Font & window size|
+|<br>Explanation of typing|
+|<br>Embraces & uses mistakes|
 |<br>Use of notes on paper/tablet|
-|<br>Avoid Being Disturbed|
+|<br>Distraction-free screen|
+|<br>Dismissive language|
```

Comment:
Perhaps remove the word 'dismissive'.  It is the only negative word and really stands out. The other elements are all neutral or positively worded

Reply:
@lexnederbragt would "dismissive-free language" work here, mirroring the criterion above? This piece also doesn't have a clear positive in the elaborated rubric, and I'm struggling to come up with one. We don't need to substitute dismissive language with mandatory positives, we need clear speech that lacks these kinds of "helpful" (but not actually helpful) decorations. Substitutions with more effectively encouraging phrases are at the instructor's discretion and don't really need to be scored.

Reply:
What about 'Wording' - which can be scored as positive and negative during the teaching demo?